README.md

```diff
@@ -20,6 +20,16 @@ Then, update your dependencies:
 $ mix deps.get
 ```
 
+Make sure to tell Hunter which API client to use in your config:
```

Comment:
What about this?

## Application configuration

The following configuration must be set via config files (e.g., `config/config.exs`) before the `:hunter` application is used:

  * `hunter_api` - specifies the API client to use. Defaults to: `Hunter.Api.HTTPClient`. See the "Clients" section in the `Hunter.Client` module for more information.

For example, to configure the `:hunter_api` options in a `config/config.exs` file:

```elixir
config :hunter,
  hunter_api: Hunter.Api.HTTPClient
```